Makefile

```diff
@@ -0,0 +1,32 @@
+run: start sleep setup
+start:
+	docker-compose up -d
+	echo -e "\033[0;32m Inspire Started. \033[0m"
+
+stop:
+	docker-compose down
+	kill -9 $(lsof -ti:8080)
+	kill -9 $(lsof -ti:8000)
+	kill -9 $(lsof -ti:3000)
+	kill -9 $(lsof -ti:4572)
+	kill -9 $(lsof -ti:4566)
+	kill -9 $(lsof -ti:6379)
+	kill -9 $(lsof -ti:5432)
+	echo -e "\033[0;32m Inspire Stopped. \033[0m"
+
+sleep:
+	sleep 10
+
+ui:
+	docker-compose up -d ui
+	sleep 5
+
+setup:
+	docker-compose -f docker-compose.yml exec hep-web ./scripts/setup
+	docker-compose -f docker-compose.yml exec next-web inspirehep db create
```

Comment:
I would put next setup in a separate command since we're not using next that often I think